angular-workspace/projects/ni/nimble-angular/src/directives/combobox/nimble-combobox-control-value-accessor.directive.ts

```diff
@@ -158,4 +203,14 @@ export class NimbleComboboxControlValueAccessorDirective implements ControlValue
     private setProperty(key: string, value: unknown): void {
         this._renderer.setProperty(this._elementRef.nativeElement, key, value);
     }
+
+    private readonly mutationCallback = (mutations: MutationRecord[]): void => {
+        const optionTextUpdate = mutations.find(mutation => {
+            return ((mutation.type === 'characterData' && mutation.target.parentElement instanceof ListOption)
```

Comment:
I'm really sorry but I can't remember. What cases does the mutation observer handle that the add, remove, and queue update in combobox list option don't handle? Shouldn't that handle all option text content updates and list item children changes?

Maybe we just need the map updates but not the mutationobserver stuff?

Reply:
Yeah this was just being done in case someone messed with the `textContent` of the native ListOption direction. We agreed this is overkill so I removed this code. This required a slight change to the `addOption` logic to only add the option to the set being tracked if it didn't already exist.